core/code/portal_detail_display_tools.js

```diff
@@ -2,6 +2,23 @@
 // hand any of these functions the details-hash of a portal, and they
 // will return pretty, displayable HTML or parts thereof.
 
+window.getPortalHistoryDetails = function (d) {
+  if (!d.history) {
+    return ('<div style="text-align: center; color: #ffce00">History missing</div>')
+  }
+
+  var visited = d.history.visited;
+  var captured = d.history.captured;
+  var scouted = d.history.scanned;
+  var colors = {true:'#03fe03',false:'#ff4a4a'};
+
+  return ('<div style="text-align: center; color: #ffce00">History:'
+  + ' <span id="history_visited" style="color:'+ colors[visited] + '">visited</span> |'
+  + ' <span id="history_captured" style="color:'+ colors[captured] + '">captured</span> |'
+  + ' <span id="history_scanned" style="color:' + colors[scouted]  + '">scouted</span>'
```

Comment:
```suggestion
  + ' <span id="history_scoutControlled" style="color:' + colors[scoutControlled]  + '">scout controlled</span>'
```

Reply:
Attention!
At the moment 'scanned' layer is wrongly titled as 'Scout Controlled' in stock intel!
"Scout Controller" is a volatile status, that the agent looses sooner or later on interaction of other agents with the same portal.
"scanned" is a permanent status, set when the agent submits the first 3D-Scan of a portal.
@MysticJay

Reply:
Better have this as comment in source code

Reply:
![image](https://user-images.githubusercontent.com/1547509/107243300-dfa7f900-69fa-11eb-94cf-00ed73da6fb4.png)

Scout Controller is a volatile status.
Scout Controlled is a historical statement.

Captured does not mean you *currently* hold the portal.  Visited does not mean you're *at* the portal.  Scout Controlled means that at some time you have been Scout Controller.

Scanning directly is irrelevant to the historical status.  Scanning a portal does not award you Scout Controller, or the Scout Controlled historical status.  Exceeding the number of scans of another person does.